app/src/main/java/com/greenfox/kalendaryo/adapter/PagerAdapter.java

```diff
@@ -30,9 +29,6 @@ public Fragment getItem(int position) {
             case 1:
                 AccountsFragment tab2 = new AccountsFragment();
                 return tab2;
```

Comment:
Could use inline return, so instead of 
```
AccountsFragment tab2 = new AccountsFragment();
return tab2;
```

use ` return new AccountsFragment()`

Reply:
also do this for the other case